src/components/calendar/CalendarWrapper.js

```diff
@@ -0,0 +1,59 @@
+import styled from "@emotion/styled";
+
+export const CalendarWrapper = styled.div`
```

Comment:
This is such a cool addition. So odd that we had to implement styling this way but seems cool!